Dockerfile

```diff
@@ -32,14 +34,21 @@ COPY scripts/bootstrap.py /usr/local/jenkins/bin/bootstrap.py
 COPY conf/jenkins/config.xml "${JENKINS_STAGING}/config.xml"
 COPY conf/jenkins/jenkins.model.JenkinsLocationConfiguration.xml "${JENKINS_STAGING}/jenkins.model.JenkinsLocationConfiguration.xml"
 COPY conf/jenkins/nodeMonitors.xml "${JENKINS_STAGING}/nodeMonitors.xml"
-COPY conf/tomcat/server.xml "${CATALINA_HOME}/conf/server.xml"
-COPY conf/tomcat/Catalina/localhost/rewrite.config "${CATALINA_HOME}/conf/Catalina/localhost/rewrite.config"
 
-RUN apt-get update
-RUN apt-get install -y git python zip jq
 RUN /usr/local/jenkins/bin/plugin_install.sh "${JENKINS_STAGING}/plugins"
 
 # Override the default property for DNS lookup caching
 RUN echo 'networkaddress.cache.ttl=60' >> ${JAVA_HOME}/jre/lib/security/java.security
 
-CMD /usr/local/jenkins/bin/bootstrap.py && catalina.sh run
+CMD /usr/local/jenkins/bin/bootstrap.py && nginx && \
+java -Dhudson.udp=-1                                \
+    -Djava.awt.headless=true                        \
+    -DhudsonDNSMultiCast.disabled=true              \
+    -jar ${JENKINS_WAR}                             \
+    --httpPort=8080                                 \
+    --webroot=JENKINS_WAR                           \
```

Comment:
The value for `--webroot` is the path that Jenkins will use for exploding the contents of its war, not the war itself. Previously this would have been somewhere in `$CATALINA_HOME` (e.g. /usr/local/tomcat). It's probably safe to keep this inside the container and not in the Jenkins data volume, since the user should (hopefully) only upgrade Jenkins when they receive a new package from us.